src/pybind/mgr/dashboard_v2/services/__init__.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from six import add_metaclass
+
+
+class ServiceMeta(type):
+    @property
+    def mgr(cls):
+        """
+        :return: Returns the MgrModule instance of this Ceph dashboard module.
+        """
+        return cls._mgr_module
+
+    @mgr.setter
+    def mgr(cls, value):
+        """
+        :param value: The MgrModule instance of the Ceph dashboard module.
+        """
+        cls._mgr_module = value
+
+
+@add_metaclass(ServiceMeta)
+class Service(object):
```

Comment:
Why does the `Service` class gets a meta class? Wouldn't the service class work exactly the same without it?

Reply:
It works, this is just to make explicitly in the code that the Service class has a `mgr` class and instance attributes.

Reply:
What's the difference between `Service` and `BaseController`, except for the name?